docs/pages/faq.mdx

```diff
@@ -123,6 +123,18 @@ Teleport provides security-critical support for the current and two previous rel
 See our [Upgrading](./setup/operations/upgrading.mdx) guide for more
 information.
 
+## Which operating systems are supported?
+
+Teleport runs on Linux, macOS, and Windows (client only).
+
+|Operating System| Version       |
+|----------------|---------------|
+| macOS          | X             |
+| Windows        | Y             |
```

Comment:
@timothyb89 What's the minimum version of Windows needed by `tsh.exe`?

Reply:
Windows 10, revision 1607 (the 2016 Anniversary update) and later.